src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int AMOUNT_BALLS = 3;
+
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
```

Comment:
According to the checklist, don't create several instances of the same class if you can use only one instance for your purposes. However, in this case, the code is correct as it creates only one instance of the `Lottery` class, which is then used in the loop.